operational_analysis/methods/turbine_long_term_gross_energy.py

```diff
@@ -109,14 +94,24 @@ def __init__(self, plant, UQ = False, num_sim = 2000, reanal_subset = ['erai', '
         
 
     @logged_method_call
-    def run(self, enable_plotting = False,
-            plot_dir = None):
+    def run(self,reanal_subset = ['erai', 'ncep2', 'merra2'],
+                 uncertainty_scada=0.005, wind_bin_thresh=(1, 3), 
+                 max_power_filter=(0.8, 0.9), correction_threshold=(0.85, 0.95),
+                 enable_plotting = False,
+                 plot_dir = None):
```

Comment:
How are wind_bin_thresh, max_power_filter, and correction_threshold specified is UQ=False? As scalars? Also, please update the docstring to explain the inputs for the UQ and non-UQ cases.

Reply:
Yes, in the UQ = False case, these are single scalar values. I have added this to the docstring description. I am just not sure how to incorporate this in the :obj: part... as we could have both tuples and floats...